packages/convenient_widgets/lib/convenient_widgets.dart

```diff
@@ -1,7 +1,6 @@
 /// Convenient widgets
 library;
 
-export 'src/adaptive_bottom_bar.dart';
```

Comment:
Removed because it is unnecessary 
- I use NavigationBar more often than BottomNavigationBar
- I have no examples of using this widget.